ui/src/app/settings/components/project-sync-windows-edit/project-sync-windows-edit.scss

```diff
@@ -16,4 +16,14 @@
     form > .row > .columns {
         padding-left: 0;
     }
-}
\ No newline at end of file
+}
+
+.options-wrapper {
```

Comment:
Argo CD uses https://css-tricks.com/bem-101/ methodology to avoid conflics in CSS. Can you please prefix `options-wrapper` and `text-wrapper` with `project-sync-windows-panel__` ? E.g.: 

```
.project-sync-windows-panel {
...
  &__options-wrapper {
  ...
  }
}
```

Reply:
@alexmt thanks for the review. Here's the screenshot for different screen widths.
![Screen Shot 2021-09-29 at 2 14 01 PM](https://user-images.githubusercontent.com/26255262/135325798-dad43d11-ab21-4b6a-98e9-71f6f326355a.png)
![Screen Shot 2021-09-29 at 2 14 47 PM](https://user-images.githubusercontent.com/26255262/135325845-515e8116-4a5f-4e50-9e70-5e444d813a72.png)
![Screen Shot 2021-09-29 at 2 14 31 PM](https://user-images.githubusercontent.com/26255262/135325865-b7c74d8b-6cad-4673-bd1f-768ac509975f.png)

Let me know how you think about it, I will make an update addresses the class name fix soon.

Reply:
Thanks for attaching screenshot! It looks good. Options are not cropped anymore so I think it solves the issue reported in #5261